app/models/grouping.rb

```diff
@@ -229,5 +229,7 @@ def accept_tracker_story
 
     story.current_state = "accepted"
     story.save
+
+    rescue
```

Comment:
Yeah, I'm gonna try to implement the "bonus" task in the PT story to use the current user's API key. Figured that should be a separate commit.